providers/base/bin/graphics_env.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
-# This script checks if the submitted VIDEO resource is from AMD and if it is
+# This script checks if the submitted VIDEO resource is from AMD or nvidia and if it is
 # a discrete GPU (graphics_card_resource orders GPUs by index: 1 is the
 # integrated one, 2 is the discrete one).
 #
 # This script has to be sourced in order to set an environment variable that
-# is used by the open source AMD driver to trigger the use of discrete GPU.
+# is used by the open source AMD driver and properties nvidia driver to trigger the use of discrete GPU.
```

Comment:
I think these lines should be truncated at 80 characters. Not sure if that's the case after your additions.

Reply:
Seems like shellcheck won't check this. I'm going to fix it.